app/concerns/agent_controller_concern.rb

```diff
@@ -55,6 +55,9 @@ def control!
           when 'enable'
             case
             when target.disabled?
+              if options['drop_pending_events']
```

Comment:
```suggestion
              if boolify(interpolated['drop_pending_events'])
```

Using `interpolated` allows the user to use Liquid in the option and `boolify` ensure the value is set to `true` and not anything that is truthy in Ruby.